plane/source/docs/guide-vtail-plane.rst

```diff
@@ -0,0 +1,163 @@
+.. _guide-vtail-plane:
+
+============
+VTail Planes
+============
+
+A common alternative to an elevator and rudder is a VTail, or
+sometimes an ATail (which is an upside down VTail).
+
+A VTail aircraft has the same functionality as a normal four channel
+plane, but needs special care for the setting up of the VTail servos.
+
+A typical VTail setup is
+
+- servo output 1 is aileron
+- servo output 2 is left vtail servo
+- servo output 3 is throttle
+- servo output 4 is right vtail servo
+
+You don't have to use this ordering if it isn't convenient for your
+plane.
+
+.. warning:: You should remove the propeller from your aircraft before
+             starting your setup.
+
+Setting Up Your Plane
+=====================
+
+After you have setup your :ref:`RC inputs <rc-throw-trim>`, the next
+step is to setup your 4 outputs.
+
+You can connect your 4 servo cables to any output of your autopilot,
+although using the defaults listed above is recommended.
+
+Next check that the SERVOn_FUNCTION values are correct. The following
+table shows the right settings for the default output ordering for a
+VTail plane.
+
+.. raw:: html
+
+   <table border="1" class="docutils">
+   <tr><th>Parameter</th><th>Value</th><th>Meaning</th></tr>
+   <tr><td>SERVO1_FUNCTION</td><td>4</td><td>aileron</td></tr>
+   <tr><td>SERVO2_FUNCTION</td><td>79</td><td>left vtail</td></tr>
+   <tr><td>SERVO3_FUNCTION</td><td>70</td><td>throttle</td></tr>
+   <tr><td>SERVO4_FUNCTION</td><td>80</td><td>right vtail</td></tr>
+   </table>
+
+Servo Reversal
+==============
+
+The next step is to get the reversals right. You should connect the
+battery (with propeller removed) and turn on your RC transmitter. Now
+switch to MANUAL mode and disable the output safety (if enabled).
```

Comment:
to "disable" the output safety, do they simply press the button? I think I know what you're talking about but the terminology comes across odd.